packages/toolkit/scripts/project/bash/init.sh

```diff
@@ -0,0 +1,93 @@
+# Create dir if it does not exist
+if [ ! -d "$init_path" ]; then
+	mkdir -p "$init_path"
+fi
+
+# Check if init_path direcfory is not empty
+if [ $(ls -A "$init_path" | wc -l) -gt 0 ]; then
+	echo "Directory $init_path is not empty. Please provide an empty directory to initialize the project."
+	exit 1
+fi
+
+
+# If template is not empty, git clone template to init_path
+if [ ! -z "$template" ]; then
+	git clone "$template" "$init_path"
+	rm -rf "$init_path/.git"
+fi
+
+# Fixes weird sed error
+LANG=C
+
+# Replace TenUpTheme in all files inside init_path with $name_camel_case recursively
+find "$init_path" -type f -exec sed -i '' -e "s/TenUpTheme/${project_name_camel_case}Theme/g" {} \;
```

Comment:
The sed commands here doesn't seem to be working with GNU SED cc @dustinrue . I get "sed: can't read : No such file or directory"